ludwig/schema/metadata/configs/combiners.yaml

```diff
@@ -1,4 +1,16 @@
 ComparatorCombiner:
+    type:
+        short_description:
+            Compares the representation of two defined entities.
```

Comment:
Appropriate for recommendation problems where there are features are associated with distinct entities, and the output depends on entity-level comparison.